core/src/main/java/bisq/core/btc/wallet/BtcWalletService.java

```diff
@@ -1278,18 +1279,18 @@ public Transaction createRefundPayoutTx(Coin buyerAmount,
             throws AddressFormatException, InsufficientMoneyException, WalletException, TransactionVerificationException {
         Transaction tx = new Transaction(params);
         Preconditions.checkArgument(buyerAmount.add(sellerAmount).isPositive(),
-                "The sellerAmount + buyerAmount must be positive.");
+                Res.get("validation.refundPayoutTx.buyerSellerNotPositive"));
         // buyerAmount can be 0
         if (buyerAmount.isPositive()) {
             Preconditions.checkArgument(Restrictions.isAboveDust(buyerAmount),
-                    "The buyerAmount is too low (dust limit).");
+                    Res.get("validation.amountBelowDust.shortVar", "buyerAmount"));
```

Comment:
This kind of argument defeats the aim of making the strings translatable. In this case it's better to just make two strings, or reuse `validation.amountBelowDust.short`